src/lib/date-picker/calendar-table.html

```diff
@@ -0,0 +1,37 @@
+<table class="md-calendar-table-table">
+  <!-- If there's not enough space in the first row, create a separate label row. -->
+  <tr *ngIf="_firstRowOffset() < labelMinCells">
+    <td class="md-calendar-table-label" colspan="7" >{{label}}</td>
+  </tr>
+
+  <!-- Create the first row separately so we can include a special spacer cell. -->
+  <tr *ngFor="let row of data | slice:0:1">
+    <td *ngIf="_firstRowOffset()"
```

Comment:
Have you considered expanding the ngIf to check the index? That way, you can avoid duplicating the whole ngFor block.

```html
<tr *ngFor="let row of data; let i=index">
   <td *ngIf="index === 0 && _firstRowOffset()"
```